Wrapping/Generators/Python/Tests/extras.py

```diff
@@ -200,6 +200,10 @@ def custom_callback(name, progress):
 itk.meshwrite(mesh, sys.argv[5], compression=True)
 
 # smoke test wasm / Python / NumPy conversion
+
+image_dict = itk.dict_from_image(image)
+image_back = itk.image_from_dict(image_dict)
```

Comment:
For improved coverage would it make sense to add a couple of basic checks to validate that `image` and `image_back` metadata are equivalent?

Reply:
Yes, I will add a call to `itk.comparison_image_filter`.

Reply:
Looks great, thanks.

On review my understanding is that [itk::Testing::ComparisonImageFilter](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Core/TestKernel/include/itkTestingComparisonImageFilter.h) directly compares image buffers and does not validate that metadata is equivalent between two images. For better coverage overall, is there a filter that does both?

Validating buffers will cover the `SetRegions` fix introduced in this PR, so for the purposes of this individual fix it is fine to move forward with `ComparisonImageFilter`.

Reply:
ComparisonImageFilter checks metadata per: https://github.com/InsightSoftwareConsortium/ITK/blob/d007ee8e78ab367d10a4fc116fceb4ba07604174/Modules/Core/TestKernel/include/itkTestingComparisonImageFilter.hxx#L48

Reply:
I missed that, thanks!